### DISCUSSION TITLE
what is Preprocessing samples? What is Max text length? Min text length?

### DISCUSSION THREAD

**Discussion by omkarade** [coqui-ai/TTS#1745] [General Q&A]
TRAINING (2022-07-17 19:08:43) 
 | > Preprocessing samples
 | > Max text length: 3
 | > Min text length: 3
 | > Avg text length: 3.0

**Comment by omkarade** [comment]
can please someone help me to understand those Parameter.

**Reply by erogol** [reply]
these are min max avg text lengths of your dataset.
